### PR TITLE
Remove core from asp.net in migration

### PR DESCRIPTION
**Pull request by adegeo**
## Summary

The migration guide entry for the upgrade tool isn't for ASP.NET Core, it's for ASP.NET

### CODE REVIEW

File: docs/fundamentals/toc.yml [hunk 1]
```diff
@@ -2316,7 +2316,7 @@ items:
         href: ../core/porting/upgrade-assistant-wpf-framework.md
       - name: Windows Forms
         href: ../core/porting/upgrade-assistant-winforms-framework.md
-      - name: ASP.NET Core
+      - name: ASP.NET
```
Comment (Youssef1313) [inline docs/fundamentals/toc.yml:2319]: The parent toc node says: "Port from .NET Framework".
So I was expect the "Core" suffix is there to mean this is migration **to** (not from) ASP.NET Core, which is correct.

Comment (adegeo) [reply docs/fundamentals/toc.yml:2319]: I'm still redoing the entire migration section, but no, the articles are focused on migrating a **type of project** to .NET 5.

**Review by BillWagner — APPROVED** [review]
This LGTM @adegeo 

You can :shipit: when ready.

**Comment by BillWagner** [comment]
closing and reopening for a fresh build

**Pull request merged**